### DISCUSSION TITLE
Alsavo Heat Pump control application - Homey Pro

### DISCUSSION THREAD

**Discussion by vonbarner** [strandborg/AlsavoCtrl#26] [Q&A]
Hello Stranborg.

Do you have any experience implementing The pool heating inverter into Homey Pro?

Michael

**Comment by strandborg** [comment]
Unfortunately no, my own setup is running on OpenHAB

**Reply by vonbarner** [reply]
Thank you for your response. I'm totally at a loss as to how to get started and put it into Homey. But couldn't it be thought that if I try to follow that description, I might get a bit off track? 😊 I have no knowledge whatsoever of the programs you must use as auxiliary programs. But I just have to try my hand at it.
